src/ppx/xr_component.cpp

```diff
@@ -106,18 +137,32 @@ void XrComponent::InitializeBeforeGrfxDeviceInit(const XrComponentCreateInfo& cr
     instanceCreateInfo.enabledApiLayerCount       = (uint32_t)xrInstanceLayers.size();
     instanceCreateInfo.enabledApiLayerNames       = xrInstanceLayers.data();
     instanceCreateInfo.applicationInfo.apiVersion = XR_CURRENT_API_VERSION;
+
+#if defined(PPX_ANDROID)
```

Comment:
Should this also check defined(PPX_OPENXR) if it's only relevant for OpenXR and the extension is only enabled if PPX_OPENXR is true above ?

Reply:
This is needed for all mobile OpenXR implementations, including Quest. PPX_ANDROID should be sufficient.

Reply:
But makes sense not to enable it on Android, as android can be non-openxr. Fixed.

Reply:
Upon closer look, would this code even run in a non-openxr case? It looks like it should only run if XR is enabled?